src/components/core/ApplicationHeader.tsx

```diff
@@ -161,10 +162,10 @@ export default function ApplicationHeader({ opened, setOpened, theme }: Applicat
 						</div>
 
 						{/* Report Problem */}
-						<div>
-							<Link href="https://forms.office.com/r/z48ExG8dPs">
-								<UnstyledButton className="w-40 h-full p-2">Report Problem</UnstyledButton>
-							</Link>
+						<div className="ml-3">
```

Comment:
looks cool cool, just one thing I want to ask to (add) na!
I think it's not responsive. or it is just on my device.